pkg/ocicni/ocicni.go

```diff
@@ -248,10 +245,8 @@ func initCNI(exec cniinvoke.Exec, cacheDir, defaultNetName string, confDir strin
 		return nil, err
 	}
 
-	startWg := sync.WaitGroup{}
```

Comment:
The waitgroup doesn't do anything here, since monitorConfDir calls startWg.Done() without a defer. Line 148

Reply:
@rphillips the waitgroup for monitorConfDir() is actually to ensure that the plugin initialization waits until the goroutine is *started*, not that it is done. We want to ensure that as much as possible of the plugin setup is running before returning from init to the runtime.